conf.d/main

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh -ex
```

Comment:
I see you pulled out all the conf.d scripts and put most of the "stuff" in the main. Reason for this?